tutorials/nservicebus-sagas/2-timeouts/Snippets/Core_7/BuyersRemorseTimeoutClassDefinition/BuyersRemorseTimeoutClassDefinition.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Core_7.BuyersRemorseTimeoutClassDefinition
+{
+    #region BuyersRemorseTimeoutClassDefinition
+
+    class BuyersRemorsePolicy
+        : Saga<BuyersRemorseState>
```

Comment:
And here... I see a pattern, but I dislike the breaking up continuously. I prefer the first thing to be on the same line.